components/Roadmap/index.jsx

```diff
@@ -52,25 +67,49 @@ const RoadmapPage = () => {
               />
             </Col>
             <Col xs={24} md={24} lg={14}>
+            <Popover 
+              content={<>
+                {getTagItems(item.tag)[1]}
+                  <>
```

Comment:
feels like we can avoid this Fragment here?

Reply:
Removed it now!